packages/api/tests/endpoints/collections/create-collection.js

```diff
@@ -180,9 +211,21 @@ test('POST creates a new collection in all data stores', async (t) => {
       version: newCollection.version,
     }
   ));
+
+  const { Messages } = await sqs().receiveMessage({
+    QueueUrl: t.context.QueueUrl,
+    WaitTimeSeconds: 10,
+  }).promise();
+
+  t.is(Messages.length, 1);
+
+  const message = JSON.parse(JSON.parse(Messages[0].Body).Message);
+  const omitFields = ['createdAt', 'updatedAt'];
+  t.is(message.event, 'Create');
+  t.deepEqual(omit(message.record, omitFields), omit(newCollection, omitFields));
```

Comment:
I think we want to compare against `fetchedDynamoRecord`, not `newCollection`. The intent here is to prove that what we send to the SNS topic represents the data after we have written to the database, not before. By comparing against `newCollection`, we're testing against the record before it is written to the database. These two things may often be the same, but not always

Reply:
Good point

Reply:
[`889e8be` (#2432)](https://github.com/nasa/cumulus/pull/2432/commits/889e8be4a4de6901e7e3a1adb756dfbe44fd1c00)